src/components/connection_handler/src/connection_handler_impl.cc

```diff
@@ -874,6 +874,18 @@ void ConnectionHandlerImpl::CreateWebEngineDevice() {
   transport_manager_.CreateWebEngineDevice();
 }
 
+bool ConnectionHandlerImpl::IsAppPresentInConnections(
```

Comment:
@ydementieiev From my point here is not much about an application but a connection. I'd suggest something like `DoesConnectionExist` instead. I know it's ugly enough but relevant.

Reply:
@mvorobio renamed